### DISCUSSION TITLE
Visual improvement at the percentage level

### DISCUSSION THREAD

**Discussion by maxrodro** [Mimetis/Dotmim.Sync#728] [Q&A]
Hi @Mimetis 

For a future version it would be good to only put two decimals, so that it maintains a standard

![image](https://user-images.githubusercontent.com/64550772/165598187-a8f88107-867c-476b-b9dc-3453a4db8478.png)

**Comment by Mimetis** [comment]
You can use the Percentage Formatting trick `$"{MyDecimalValue:p"` (using the `:p`)

Here is an example I usually use:
``` cs
    var progress = new SynchronousProgress<ProgressArgs>(s =>
    {
        Console.ForegroundColor = ConsoleColor.Green;
        Console.WriteLine($"{s.ProgressPercentage:p}:  \t[{s.Source[..Math.Min(4, s.Source.Length)]}] {s.TypeName}:\t{s.Message}");
        Console.ResetColor();
    });
```

**Reply by maxrodro** [reply]
Hi @Mimetis 

What a great trick thank you very much
